cockatrice/src/chatview.cpp

```diff
@@ -374,9 +417,27 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
                 while (fullMentionUpToSpaceOrEnd.size());
             }
         }
+        else if (index == highlightWordFirstIndex)
+        {
+            // You have received a valid mention of custom word!!
+            int firstSpace = (message.indexOf(" ") == -1 ? message.size() : message.indexOf(" "));
+            mentionFormat.setBackground(QBrush(getCustomMentionColor()));
```

Comment:
Mention color is to you with YOU and your USERNAME. It is nothing to do with search terms. It should be different from the base mention color. You are looking at it in a single use case, which is adding common uses of your own name. Think about it in general use. having the same color add confusion.